alerts.yml

```diff
@@ -22,8 +22,8 @@ spec:
              sum(max_over_time(logd_messages_total{log_app="su-se-bakover",log_level="Error"}[15m]
              offset 15m) or vector(0)) > 0'
       for: 1s
-      description: "su-se-bakover har fått en ny error i loggen siste 5 min"
-      action: "Det er logget en melding i Kibana. Sjekk logger her: `https://logs.adeo.no`"
+      description: "su-se-bakover har fått en ny error i loggen siste 15 min"
+      action: "Det er logget en melding i Kibana. Sjekk logger her: `https://logs.adeo.no/goto/10065d0eff34c88407beb46993e0d729`"
```

Comment:
Denne kan du legge som en VAR hvis det er så om å gjøre å spesifisere et søk

Reply:
rettet bare til 15 min, fordi at det er 15 min det søkes på i spørringen. Var bare sånn at det skulle bli likt. Kan godt være 5 min i stedet begge steder. Kanskje det er bedre?

Reply:
Tror han tenkte på URLen :) 
Vi kan sannsynligvis sende den inn fra alerts-deploy.yml sånn som SLACK_ALERTS_CHANNEL: `VAR: SLACK_ALERTS_CHANNEL=#supplerende_stonad_alerts_dev`
Hvis det funker så slipper vi vel to filer, antar man kan bruke samme fil som RESOURCE?